templates/userprofile/profile_page.html

```diff
@@ -7,19 +7,25 @@
 <div class="d-flex flex-column align-items-center">
     <h1 style="color: white;">{{ user.username }}'s' Profile Page</h1>
 
-    <div class="content-section" style="background-color: white">
+    <div class="content-section" style="padding:1%; background-color: white">
         <div class="media">
             <img class="rounded-circle account-img" src="{{ user.profile.image.url }}">
         </div>
 
         <div>
             <h4 class="border-bottom mb-4">{{ user.first_name }} {{ user.last_name }}</h4>
             <p>Username: {{ user.username }}</p>
-            <p>Email: {{user.email}}</p>
-            <p class="border-bottom mb-4">Interests</p>
-            <p>ADD CODE TO DISPLAY ACTIVITES</p>
+            <p>Email: {{ user.email }}</p>
+            <p>Location: {{ user.profile.location }}</p>
+            <p>Willing to Travel To: {{ user.profile.distance }}</p>
+            <p class="border-bottom mb-4">Interests:</p>
+            {% for key, value in profile_dic.items %}
+                {% if value == True %}
+                    {{ key }}  
+                {% endif %}
+            {% endfor %}
```

Comment:
This works great, but it's a little hard to read without distinction between the different sports. I am wondering if we could display each item as a list item instead. Something like the code below should work for an unordered list. Let me know what you think!

        <ul>
            {% for key, value in profile_dic.items %}
                {% if value == True %}
                   <li> {{ key }}  </li>
                {% endif %}
            {% endfor %}
        </ul>